src/index.js

```diff
@@ -13,6 +13,9 @@ function replaceExt(p, ext) {
 const defaultBabelExtensions = ['.js', '.jsx', '.es', '.es6'];
 
 export function mapModule(source, file, pluginOpts, cwd) {
+    // regardless of cwd option, file param is passed relative to the working directory
+    file = path.resolve(file);
```

Comment:
Sorry I was not clear.

That's because `file` is a relative path from babel process (relative to the real current working directory, not the option `cwd`). Then later in `mapToRelative` it tries to resolve `currentFile` (`file`) with that custom `cwd`, leading to a invalid path in case you're running the command somewhere else. So this `path.resolve()` is needed to obtain the right absolute path of the file.

My english writing skills are not so good yet, I guess. So maybe you could help with a better comment.